motionplan/armplanning/cmd-plan/cmd-plan.go

```diff
@@ -200,23 +217,20 @@ func visualize(req armplanning.PlanRequest, plan motionplan.Plan, mylog *log.Log
 			}
 
 			for _, mp := range midPoints {
-				err := drawPosition(req, mp)
-				if err != nil {
+				if err := viz.DrawFrameSystem(req.FrameSystem, mp); err != nil {
 					return err
 				}
 
-				time.Sleep(renderFramePeriod)
+				time.Sleep(renderFramePeriod / time.Duration(len(midPoints)))
```

Comment:
so actually, don't think we should divide here, just make the time we sleep smaller
because of midpoints, each draw is about the same time in the real world.
so maybe should just make it 5ms